### PR TITLE
[NO GBP] Maintainer requested changes to the locker/closet grammar PR

### PR DESCRIPTION
**Pull request by NamelessFairy**
## About The Pull Request

San requested changes to #73331 however the PR got merged before I had an opportunity to update my PR, this PR should contain the requested changes/reverts.
## Why It's Good For The Game

Should have been in the previous PR but was not.
## Changelog
:cl:
spellcheck: Head of staff lockers and prison cell lockers have had their proper noun status returned.
/:cl:

### CODE REVIEW

**Comment by NamelessFairy** [comment]
@san7890

File: code/game/objects/structures/crates_lockers/closets/secure/scientist.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /obj/structure/closet/secure_closet/research_director
-	name = "research director's locker"
+	name = "Research Director's locker"
```
Comment (Mothblocks) [inline code/game/objects/structures/crates_lockers/closets/secure/scientist.dm:2]: i really don't think you should change any of these that are referring to a job. we don't do this normally and it looks terrible

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by Mothblocks** [comment]
> but we should really change the unique lockers that there's only one of per station (e.g. Quartermaster's locker, Chief Engineer's locker are both specific and unique) stylized like that

i don't think i agree @san7890

File: code/game/objects/structures/crates_lockers/closets/secure/scientist.dm (hunk 1 above)
Comment (Mothblocks) [reply code/game/objects/structures/crates_lockers/closets/secure/scientist.dm:2]: ah this was san's complaint, i did not realize. i thought he was only concerned about the cells

**Comment by Mothblocks** [comment]
especially since this will read as "You hit Captain's locker", and changing to add `\improper` goes against what you want wrt these being treated as proper objects

**Comment by Time-Green** [comment]
I’m dumb thank you

**Comment by san7890** [comment]
> i don't think i agree, we don't do this with job outfits/medals/similar and we shouldn't because it looks ugly compared to everything else @san7890

yeah i forgot we already had the precedent set with clothes and medals and the likes i was just forgetful

**Pull request merged**